utility/src/main/java/org/oppia/util/parser/ImageLoader.java

```diff
@@ -0,0 +1,17 @@
+package org.oppia.util.parser;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.target.SimpleTarget;
+
+public interface ImageLoader {
```

Comment:
Can we convert this java file to kotlin?

Reply:
+1

No Java files please. :) Unless we need to, in which case let's chat about it.

Reply:
Fixed it :)